en/lesson-8.md

```diff
@@ -1,3 +1,156 @@
 ---
 title: Tables
 ---
+
+# Basic Tables
+
+Tables in LaTeX are set using the `tabular` environment. In order to typeset a
+`tabular` we have to tell LaTeX how many columns will be needed and how they
+should be aligned. This is done in a mandatory argument &ndash; often referred
+to as table preamble &ndash; to the `tabular` environment, in which you specify
+the columns by using single letter names. The available column types are:
+
+`l`
```

Comment:
Should we have Frank's new `w` type column here?

Reply:
@josephwright IIRC, it is contained in `array.sty` not in the kernel.

Reply:
Even though the `array` extension requires using a package I think it is is so much core that it should be recommended upfront (caveat: only commenting on the level of comments above, not yet looked at what or how somehting is described)

Reply:
@josephwright something else: The used MD-Syntax in this is for descriptions (<item>\n: <description>), but it's not part of core-markdown and I have no idea whether it'll be supported on learnlatex. It might be necessary to reformulate this as a markdown table, which are also not part of core-markdown, or an inline HTML table if absolutely necessary.

Reply:
@Skillmon  We are using GitHub Pages behind-the-scenes: whatever works in that regard is fine. (I've no desire to run Jekyll locally)